views/data-analysis/iframe-view.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+//******************************************************************************
+//                                 iframe-view.php
+// PHIS-SILEX
+// Copyright © INRA 2018
+// Creation date: 21 feb. 2019
+// Contact: arnaud.charleroy@inra.fr, anne.tireau@inra.fr, pascal.neveu@inra.fr
+//******************************************************************************
+use yii\helpers\Html;
+use kartik\icons\Icon;
+
+/* @var $this yii\web\View */
+
+$this->title = Yii::t('app', '{n, plural, =1{Stat/Vizu Application} other{Stat/Vizu Applications}}', ['n' => 2]);
+$this->params['breadcrumbs'][] = $this->title;
+echo Html::beginTag('div', ['class' => 'row']);
+echo Html::tag('h5',
+        Icon::show('flask', ['class' => 'fa-large'], Icon::FA). "Experimental version",
```

Comment:
Add translation
Show it only if there are apps

Reply:
Done